invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsField.js

```diff
@@ -27,10 +27,10 @@ const creatibutorNameDisplay = (value) => {
 
   const familyName = _get(value, "person_or_org.family_name", "");
   const givenName = _get(value, "person_or_org.given_name", "");
-  const affiliationName = _get(value, `affiliations[0].name`, "");
+  const affiliations = value?.affiliations.map((affiliation) => affiliation.name);
```

Comment:
Question: Looks like this works fine from the screenshot you have shared. Just double checking that affiliations is not silently failing in the console when it's not present, I guess it will be an empty list which is fine, otherwise we would also need to add the `?` operator.

Reply:
Nope no failure in console :)